scripts/submit_emod3d.py

```diff
@@ -27,91 +27,103 @@
 from shared_workflow.shared_template import write_sl_script
 
 
-def main(args: argparse.Namespace, logger: Logger = get_basic_logger()):
-    params = utils.load_sim_params(os.path.join(args.rel_dir, "sim_params.yaml"))
+def main(
+    auto: bool = False,
+    machine: str = host,
+    ncores: int = platform_config[const.PLATFORM_CONFIG.HF_DEFAULT_NCORES.name],
+    rel_dir: Path = Path("."),
+    retries: int = 0,
+    write_directory: Path = None,
+    logger: Logger = get_basic_logger(),
+):
+    rel_dir = rel_dir.resolve()
+    try:
+        params = utils.load_sim_params(rel_dir / "sim_params.yaml")
+    except FileNotFoundError:
+        logger.error(f"Error: sim_params.yaml doesn't exist in {rel_dir}")
+        raise
 
-    submit_yes = True if args.auto else confirm("Also submit the job for you?")
+    params.sim_dir = Path(params.sim_dir).resolve()
 
-    logger.debug("params.srf_file {}".format(params.srf_file))
-    # Get the srf(rup) name without extensions
-    srf_name = os.path.splitext(os.path.basename(params.srf_file))[0]
+    submit_yes = True if auto else confirm("Also submit the job for you?")
 
-    if args.srf is None or srf_name == args.srf:
-        logger.debug("not set_params_only")
-        # get lf_sim_dir
-        sim_dir = os.path.abspath(params.sim_dir)
-        lf_sim_dir = sim_struct.get_lf_dir(sim_dir)
+    logger.debug(f"params.srf_file {params.srf_file}")
 
-        # default_core will be changed is user passes ncore
-        nt = int(float(params.sim_duration) / float(params.dt))
+    # Get the srf(rup) name without extensions
+    srf_name = Path(params.srf_file).stem
 
-        target_qconfig = get_machine_config(args.machine)
+    logger.debug("not set_params_only")
+    # get lf_sim_dir
+    sim_dir = params.sim_dir
+    lf_sim_dir = sim_struct.get_lf_dir(sim_dir)
 
-        retries = args.retries if hasattr(args, "retries") else None
+    # default_core will be changed is user passes ncores
```

Comment:
This comment doesn't seem in line with the code around it?

Reply:
Looks like someone didn't delete the comment 3 years ago.
```
 # default_core will be changed is user passes ncore
 n_cores = args.ncore
 if n_cores != default_core:
```
Removed the comment.